MAUI/Chat/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: About .NET MAUI Chat Control | Syncfusion
+description: Learn here all about Introduction of the Syncfusion .NET MAUI Chat (SfChat) control, its elements, and more.
+platform: MAUI
+control: Chat
+documentation: ug
+---
+
+# Overview of .NET MAUI Chat (SfChat)
+
+The Syncfusion .NET MAUI Chat control offers a contemporary conversational chatbot experience, allowing for flexible customization of the layout to display conversations between two or more users.
+
+## Key features
+
+* **Message types:** Incorporated functionality for diverse message formats, including text, calendar, date picker, time picker, and clickable hyperlinks to display a range of message types.
```

Comment:
pls reduce the content of "Message Types"

Reply:
Changed the content and replaced with Xamarin content